### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by xkotori**
Fixed "DEVELOPERS_AND_CONTRIBUTORS", "CONFIRM", "CONNECTION_FAILED", "OUTPUT_DEVICE", "RESOURCE_CHECK_FAILED", "EMPTY_HERE" typos.

### CODE REVIEW

**Comment by C0kkie** [comment]
You miss a few translation, which needs to be added

**Comment by xkotori** [comment]
> You miss a few translation, which needs to be added

Ready.

File: langs/pl.json [hunk 1]
```diff
@@ -86,9 +86,26 @@
     "DELETE_DATA": "Usuń dane",
     "CONFIRM_RESOURCE_DELETE": "Czy na pewno chcesz usunąć {0} zasobów?\nZostaną one pobrane ponownie przy następnym połączeniu",
     "CONFIRM_DATA_DELETE": "Czy na pewno chcesz usunąć {0} danych serwera?\nMogą one zawierać zapisane ustawienia, dane logowania itp.",
-    "CONFIRM": "Podtwierdź",
+    "CONFIRM": "Potwierdź",
     "HELP": "Pomoc",
     "RECONNECT": "Połącz ponownie",
-    "CRASH_REPORTER_ENABLED": "Włącz usługę zgłaszania awarii"
+    "CRASH_REPORTER_ENABLED": "Włącz usługę zgłaszania awarii",
+
```
Comment (C0kkie) [inline langs/pl.json:93]: remove the empty line here

Comment (xkotori) [reply langs/pl.json]: Done

**Pull request merged**